modules/islandora_scholar_embargo/includes/embargo.inc

```diff
@@ -485,7 +488,16 @@ EOQ;
  *   including 'obj' as the object ID, 'expiry' as the expiry date, and 'date'
  *   as the item's created or last modified date.
  */
-function islandora_scholar_embargo_get_embargoed_items_subset($offset = '1900-01-01T00:00:00.000Z', $limit = 10) {
+function islandora_scholar_embargo_get_embargoed_items_subset($offset_date = '1900-01-01T00:00:00.000Z', $offset_pid = NULL, $limit = 10) {
+  // Establish some parameters for the query.
+  $filter = is_null($offset_pid) ?
+    "FILTER(?date > \"$offset_date\"^^<http://www.w3.org/2001/XMLSchema#dateTime>)" :
+    "FILTER(?date > \"$offset_date\"^^<http://www.w3.org/2001/XMLSchema#dateTime> || (?date = \"$offset_date\"^^<http://www.w3.org/2001/XMLSchema#dateTime> && str(?obj) > \"$offset_pid\"))";
+  $order_by = is_null($offset_pid) ?
+    "ORDER BY ASC(?date)" :
```

Comment:
This would still have to order `ASC(?obj)`, to ensure we can pass correctly to the second slice.